uint24.go

```diff
@@ -0,0 +1,27 @@
+// with the courtesy of this github file: https://github.com/Cryptkeeper/go-fseq/blob/v0.2.6/pkg/uint24/uint24.go
+// used to convert a int32 to uint24 (mediumint)
+package ghostferry
+
+const MaxUint24 = 1<<24 - 1
+
+func NewUint24(val int32) *Uint24 {
```

Comment:
Is this needed? Can we not just use https://github.com/go-mysql-org/go-mysql/blob/master/mysql/parse_binary.go#L22-L31

Reply:
It would take a int32 to byte conversion, which in itself would require some code which you might not want to put in that location as well